### PR TITLE
= actor fix up AbstractActor's Java doc.

### PR DESCRIPTION
**Pull request by He-Pin**

### CODE REVIEW

**Review by chbatey — APPROVED** [review]
LGTM

File: akka-actor/src/main/scala/akka/actor/AbstractActor.scala [hunk 1]
```diff
@@ -108,23 +108,21 @@ object AbstractActor {
...
- *       })
- *       .match(Integer.class, i -> {
- *         sender().tell(i * 10, self());
- *       })
- *       .match(String.class, s -> s.startsWith("foo"), s -> {
- *         sender().tell(s.toUpperCase(), self());
- *       })
- *       .build();
- *     );
- *   }
```
Comment (patriknw) [inline akka-actor/src/main/scala/akka/actor/AbstractActor.scala]: another data point against having uncompiled code in scaladoc

**Review by patriknw — APPROVED** [review]
LGTM

File: akka-actor/src/main/scala/akka/actor/AbstractActor.scala (hunk 1 above)
Comment (hepin1989) [reply akka-actor/src/main/scala/akka/actor/AbstractActor.scala]: yeah, a little misleading.

**Comment by hepin1989** [comment]
rebased

**Pull request merged**